src/components/InfiniteList/InfiniteList.tsx

```diff
@@ -107,6 +112,7 @@ export const InfiniteList = <T, C>({
       exit: (velocity) => Math.abs(velocity) < 10,
       ...props.scrollSeekConfiguration,
     },
+    overscan: 2000,
```

Comment:
Great find.

Reply:
The more and more I look at speed on React, the more I think MPDX is slow at rendering. No action item here, just writing down my thoughts.

Reply:
@dr-bizz unfortunately I don't know if adding overscan helps much. I wonder if it would be worth looking into a different Infinite scrolling option. I really don't like the user experience of this one.